changelogs/current.yaml

```diff
@@ -237,6 +237,10 @@ new_features:
   change: |
     Update ``aws_lambda`` filter to support use as an upstream HTTP filter. This allows successful calculation of
     signatures after the forwarding stage has completed, particularly if the path element is modified.
+- area: aws_lambda
+  change: |
+    Refactor aws_lambda configuration to move signer inside the config. This allows to add signature algorithm and
+    other providers configurations.
```

Comment:
I prefer if we update this to say that we are making signer part of filter setting, instead of "to move signer inside config",  which might lead to confusion for the user.

Reply:
sure, let me rewrite this description

Reply:
I don't see any configuration changes? What is the actual end user change here? Can you clarify?

/wait

Reply:
I would remove the changelog if there is no visible change in this PR.